examples/organization-level-audit-log/main.tf

```diff
@@ -2,6 +2,14 @@ provider "google" {}
 
 provider "lacework" {}
 
+variable "organization_id" {
+  default = "my-organization-id"
+}
+
+variable "project_id" {
```

Comment:
where do we use this variable?

Reply:
I need to pass this to the module, good catch